nonCisMaleInCs/DetailsView.js

```diff
@@ -0,0 +1,125 @@
+import React from 'react';
+import {StyleSheet, View, Text, Image, Button, Alert, TouchableOpacity} from 'react-native';
+// import { Icon } from 'react-native-elements' 
+// import { Header } from 'react-native-elements';
+// import Icon from 'react-native-vector-icons/FontAwesome';
+import editIcon from './edit.png';
+import nophoto from './nophoto.png';
+
+import {Colors} from 'react-native/Libraries/NewAppScreen';
+
+const onPressed = () => {
+  Alert.alert('Edit button got pressed!');
+};
+
+const DetailsView = () => {
+  return (
+    <>
```

Comment:
We will need a `ScrollView` as a wrapper to incorporate small screens too...

Reply:
Done.